colplus-admin/src/main/java/org/col/admin/task/importer/InterpreterBase.java

```diff
@@ -74,14 +82,27 @@ protected void addAndTransliterate(NeoTaxon t, VernacularName vn) {
   }
 
   protected LocalDate date(NeoTaxon t, Issue invalidIssue, Term term) {
-    return parse(DateParser.PARSER, t.verbatim.getTerm(term)).orNull(invalidIssue, t.issues);
+    Optional<FuzzyDate> date;
```

Comment:
this try catch and add issue is done with the help of the SafeParser in the "parse" method already, better reuse that and just do the if fuzzy data add partial date issue

Reply:
Yes, my problem was: the SafeParser allows for only 1 issue to be reported (INVALID_DATE) __if__ we also like to report (warning) that the date partial a.k.a. fuzzy, I couldn't do it via the SafeParser. Let me know if you think it is important that we found a fuzzy date. If not, everything can be folded back into the SafeParser again

Reply:
(I have created a new Issue constant: PARTIAL_DATE)